doc/api/esm.md

```diff
@@ -358,6 +358,9 @@ Node.js supports the following conditions:
 * `"node"` - matched for any Node.js environment. Can be a CommonJS or ES
    module file. _This condition should always come after `"import"` or
    `"require"`._
+* `"production"` - matched if `process.env.NODE_ENV` is set to `production`.
```

Comment:
i'm not clear on how this works at all. is this production for "require" or for "import"? what about for "browser" or for "node"?

Reply:
I just updated the example to a case where `production` and `browser` are used together. These conditions are fully orthogonal if we are still allowed to use that word, so by default it applies for both require and import.